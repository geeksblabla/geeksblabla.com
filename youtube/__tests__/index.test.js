const path = require("path")

const getEpisodeDetails = require("../episode-details/get-episode-details")

describe("Youtube workflow ", () => {
  test("should return the correct episode description and youtube url ", () => {
    const file = path.resolve(__dirname, "./episode.md")

    const { description, youtubeUrl } = getEpisodeDetails(file)

    console.log(description)
    expect(youtubeUrl).toBe("https://www.youtube.com/watch?v=T0ebJvUiDG4")
    expect(description).toMatchSnapshot()

    expect(true).toBe(true)
  })
})
