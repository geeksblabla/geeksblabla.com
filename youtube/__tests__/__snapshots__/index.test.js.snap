// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Youtube workflow  should return the correct episode description and youtube url  1`] = `
"In this episode, we talk about Open Source and the annual event Hacktoberfest. We answer your questions about getting started with Open Source, the best practices, and how it could help advance your career.



Guests
-------------------------------- 

Yasser Tahiri: https://www.yezz.me/

Abderrahim soubai: https://www.soubai.me/



Notes
-------------------------------- 

0:01 - Intro and welcoming.
0:04 - What is hacktoberfest
0:09 - What are the least requirements to start with Hacktoberfest and Open Source?
0:30 - What are the types of contributions?
0:40 - Are there any Moroccan repositories to contribute to?
0:44 - How to promote your Open Source project?
1:07 - Geeksblabla giveaway
1:08 - How did your contribution to Open Source help you in your career?
1:12 - Did it help you get hired easily?
1:28 - Open Source in Morocco
1:55 - Hacktoberfest participation
2:00 - Wrap up and Goodbye



Links
-------------------------------- 

Hacktoberfest 2021: https://hacktoberfest.digitalocean.com/

Awesome Morocco: https://github.com/DevC-Casa/awesome-morocco

First Contributions: https://github.com/firstcontributions/first-contributions

Hacktoberfest & Open source previous Episode: https://geeksblabla.com/blablas/hacktoberfest-open-source



Prepared and Presented by
-------------------------------- 

Youssouf EL Azizi: https://elazizi.com/



Social Media
--------------------------------

Facebook: https://www.facebook.com/geeksblabla
Instagram:  https://www.instagram.com/geeksblabla
Twitter: https://www.twitter.com/geeksblabla
Linkedin: https://www.linkedin.com/company/69317726/



Community projects :
--------------------------------

Geeksblabla:  https://geeksblabla.com/
GeeksBlaBla is a community initiative, to discuss, highlight and share the latest IT topics in Moroccan Darija.

Awesome Morocco: https://github.com/DevC-Casa/awesome-morocco
List of awesome Moroccan things for developers. Communities, Events, Blogs, Podcasts, Youtube Channels, Open Source Projects, Books and more

State Of Dev In Morocco:  https://stateofdev.ma/
In November 2020, 2287 developers from Morocco ðŸ‡²ðŸ‡¦ told us about their jobs satisfaction, salaries, and community contribution, how they learn and level up, which tools theyâ€™re using, and what they want to learn next.



You can update this description ðŸ‘‰ : https://github.com/DevC-Casa/geeksblabla.com/tree/master/blablas/__tests__/index.md"
`;
